cmd/kubeapps-apis/server/plugins.go

```diff
@@ -51,6 +48,9 @@ const (
 	clustersCAFilesPrefix   = "/etc/additional-clusters-cafiles"
 )
 
+// KubernetesClientGetter is a function type used by plugins to get a k8s client
+type KubernetesClientGetter func(context.Context) (kubernetes.Interface, dynamic.Interface, error)
```

Comment:
Created this type alias so its easier to update the signature (before I added the typed interface).